Formula/shairport-sync.rb

```diff
@@ -22,6 +27,12 @@ class ShairportSync < Formula
   depends_on "libdaemon"
   depends_on "libconfig"
 
+  # opened PR 12 Aug 2016: "use sysconfdir not $(DESTDIR)/etc for config files"
```

Comment:
Our prior `--with-configfiles=no` was a bit of a red flag that something was amiss.

Surprisingly, upstream feels the need to contemplate the matter: https://github.com/mikebrady/shairport-sync/pull/355#issuecomment-239451224